fnirsi_logger.py

```diff
@@ -96,12 +98,15 @@ def main():
     assert ep_out
 
     ep_out.write(b"\xaa\x81" + b"\x00" * 61 + b"\x8e")
-    ep_out.read(size_or_buffer=64)
+    if is_fnb58:
```

Comment:
I think this should be `if is_fnb58 or is_fnb48:`

Reply:
Probably not, I looked more closely on HID packet and determined what are first byte is, probably it is a payload type. All data packets starts from 0x04 byte, some other packets (sometimes first, sometimes not) starts from 0x03 byte and these packets not is data samples (status report?). We just can ignore all non-data packets and don't use workarounds (crutch?) such as `first_data_point`. See last commit.

Reply:
Yes, using just `0x04` is good approach.

I do not know why I didn't notice this before in my RE checks.

I am guessing that `0x03` is some device info like model name, serial number, and firmware version, etc. But it is just a guess at the moment.